Tasks/AppCenterDistributeV1/Tests/L0PublishMandatoryUpdate.ts

```diff
@@ -21,7 +21,7 @@ tmr.setInput('symbolsType', 'AndroidJava');
 tmr.setInput('mappingTxtPath', '/test/path/to/mappings.txt');
 
 process.env['BUILD_BUILDID'] = '2';
-process.env['BUILD_SOURCEBRANCHNAME'] = 'master';
+process.env['BUILD_SOURCEBRANCH'] = 'master';
```

Comment:
should this be `refs/heads/master`?

Reply:
You're correct - updated to be `refs/heads/master`.